item/historical/scripts/T007.ipynb

```diff
@@ -1,5 +1,3236 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import json\n",
+    "from pprint import pprint\n",
+    "from item.historical.scripts.util.managers.dataframe import DataframeManager\n",
+    "from item.historical.scripts.util.managers.dataframe import ColumnName\n",
+    "from item.historical.scripts.util.managers.country_code import CountryCodeManager\n",
+    "from item.common import paths"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variables used across the noteboook and not changed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET_ID = \"T007\"\n",
+    "dataframeManager = DataframeManager(DATASET_ID)\n",
+    "countryCodeManager = CountryCodeManager()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Opening the dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [
```

Comment:
Please clear outputs for these newly-added notebooks to avoid the problems mentioned above.